app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -65,4 +91,11 @@ INSTALL()
 		LUA_PACKAGE=lua
 	mv $docDir/wordgrinder/README.wg $docDir
 	rmdir $docDir/wordgrinder
+if [ -z "$secondaryArchSuffix" ]; then
```

Comment:
This will allow to have the dictionaries in the main package when we're building for a secondary arch.
```diff
-if [ -z "$secondaryArchSuffix" ]; then
 	install -d $dataDir/wordgrinder/dictionaries
 	install -t $dataDir/wordgrinder/dictionaries extras/*.dictionary
+if [ -z "$secondaryArchSuffix" ]; then
 	packageEntries dictionaries \
 		$dataDir
 fi
```
Cool, isn't it?

Reply:
So, what do we do?
Do we keep the package without the dictionaries, as I had though initially, or do we include it for wordgrinder_x86?

BTW, I realized that having  wordgrinder_dictionaries depend on wordgrinder (instead of the opposite) makes the wordgrinder_dictionaries hpkg installable only on the plafform on which it was built.

So if we wanted to be able to use wordgrinder_dictionaries on any arch, we would have to say that it does not depend on wordgrinder. Well, this can be easily done by removing the definition of REQUIRES_dictionaries.

@localanu, in case there is a consensus for including the dictionaries in wordgrinder_x86, there will be a small fix I had not forseen:
Inserting the following bloc after the definition of REQUIRES:
```diff
 	lib:libz$secondaryArchSuffix
 	"
+if [ -n "$secondaryArchSuffix" ]; then
+CONFLICTS="
+	wordgrinder_dictionaries
+	"
+fi
```
This will prevent the installation of wordgrinder_dictionaries on x86_gcc2 since on that arch the wordgrinder_x86 package will already include the dictionaries.

But please do not apply these changes before we reach a consensus on this question, as we might need to revert it.
Thanks!

Reply:
Just a thought here, then I'll silence ;)
Couldn't targetArchitecture come in handy in this case (to still be able to have a seperate package "any" on x86_gcc2), didn't check it (yet) ...

Reply:
Ok, just checked, still produces a x86 package instead of any
So I think we can be ok with this :+1: 
Good work!